agagd/agagd_core/models.py

```diff
@@ -260,6 +260,10 @@ class Meta:
 
 
 class Rating(models.Model):
+
+    # Sets a primary key.
+    id = models.BigAutoField(primary_key=True)
```

Comment:
`agagd-app-1  | WARNINGS:

agagd-app-1  | agagd_core.Rating: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.

agagd-app-1  |  HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.`

Django auto assigns a key to this model, this defines what it is to avoid future issues.

Reply:
resolved